src/elements/content-sharing/messages.js

```diff
@@ -26,6 +26,11 @@ const messages = defineMessages({
         description: 'Message that appears when the shared link in the ContentSharing Element cannot be updated.',
         id: 'be.contentSharing.sharedLinkUpdateError',
     },
+    collaboratorsLoadingError: {
+        defaultMessage: 'Could not retrieve collaborators for this item.',
+        description: 'Message that appears when the shared link in the ContentSharing Element cannot be updated.',
```

Comment:
should this description be something like "...when collaborators cannot be retrieved"?

Reply:
Good catch! Fixed.